packages/server/package.json

```diff
@@ -116,6 +116,9 @@
   "publishConfig": {
     "access": "public"
   },
+  "dependencies": {
+    "@swc/helpers": "^0.5.15"
```

Comment:
Why we need this?

Reply:
because explicit resource managament adds a bunch of code everywhere it's used

with this it'll reference a tree-shakable library instead

i don't mind either tbh but this is how i got it to work

we could also use `tslib` which has the same purpose

Reply:
i think if we set .....

https://github.com/trpc/trpc/blob/55fae79efdc0f2e911829baea2ce7228a9b457e5/scripts/getRollupConfig.ts#L100

... to `false`, we can delete this but then we'll add a bunch of noise to the bundle (which is fine) - adding `tslib`/`@swc/helpers` seems to be the recommended approach b/c then all libs "share" these additions

Reply:
#6270